policies/approval.yml

```diff
@@ -75,6 +75,12 @@ approval_rules:
       # If true, pushing new commits to a pull request will invalidate existing
       # approvals for this rule. False by default.
       invalidate_on_push: true
+      # If true, "update merges" do not invalidate approval (if invalidate_on_push
+      # is enabled) and their authors/committers do not count as contributors. An
+      # "update merge" is a merge commit that was created in the UI or via the API
+      # and merges the target branch into the pull request branch. These are
+      # commonly created by using the "Update branch" button in the UI.
+      ignore_update_merges: true
```

Comment:
I am quite sure that this will have no effect on rebase merges, where each commit is rewritten. Policy-bot is only identifying the literal merge commit from master mixed in with other PR commits, a process we avoid using and block via Flowzone.

Reply:
Are you sure? From the description [here](https://github.com/palantir/policy-bot?tab=readme-ov-file#update-merges), it may work (though I've not looked through their code).

Reply:
Yes, that is describing a merge commit (a commit with two parents) which is the same logic used for Flowzone to reject a PR for using the wrong type of master update. I am certain.